scripts/m18_submission_checker/aida_m18_submission_checker.py

```diff
@@ -0,0 +1,381 @@
+        path_items = re.split('/', name)
+
+        if name.endswith('.ttl'):
+
+            ttls_total_count += 1
+            file_dir_status = valid_dir_check(path_items)
+
+            dir = os.path.dirname(name) + '/'
+
+            if file_dir_status:
+                #get value for the current ttl dir, if it dosen't exist then add it to the dict
+                curr_val = ttl_valid_count_dict.setdefault(dir, 0)
+                curr_val += 1
+                ttl_valid_count_dict[dir] = curr_val
+
+            else:
+                #get value for the current ttl dir, if it dosen't exist then add it to the dict
+                curr_val = ttl_invalid_count_dict.setdefault(dir, 0)
+                curr_val += 1
+                ttl_invalid_count_dict[dir] = curr_val
+
+    return ttls_total_count, ttl_valid_count_dict, ttl_invalid_count_dict
+
+def do_TA1a_2_check(path_items):
+    """
+    Check the path in the archive to ensure the path is valid
+    TASK 1a Directory structure:
+    <TA1aperformer>_<run>
+        NIST
+            <document_id>.ttl                  1a: (1 to X); 2: (1)
+		INTER-TA
+		    <document_id>.ttl                  1a: (0 to X); 2: (0 or 1)
+
+
+    Ignores non ttl files.
+    Prints number of ttl files found, and number of ttl files that are in need to be checked.
+
+    :param path_items: The path to the .ttl file that needs to be checked
+    :return: 'NIST' if .ttl file is in the ../NIST/ directory, 'INTER-TA' if in the ../INTER-TA/ directory or None if not in either directory
+    :rtype: string or None
+    """
+
+    # validate that the ttl file is in an acceptable directory
+    # rule: ttl needs to be in 2nd level -- aka 3rd item in list['name', 'NIST', 'valid.ttl']
+    # rule: ttl needs to be in directory named NIST, or INTER-TA
+
+    try:
+        if ((path_items[-2] == 'NIST' or path_items[-2] == 'INTER-TA') and len(path_items) == 3):
+            return True
+    except:
+        return False
+    return False
+
+
+def do_TA1b_check(path_items):
+    """
+    Check the path in the archive to ensure the path is valid
+    Task 1b directory structure:
+    <TA1performer>_<run>
+        NIST
+            hypothesisID			(1 to Y)
+                <document_id>.ttl 		(1 to X)
+
+    Ignores non ttl files.
+    Prints number of ttl files found, and number of ttl files that are in need to be checked.
+
+    :param path_items: The path to the .ttl file that needs to be checked
+    :return: True if the file path is valid, False otherwise
+    :rtype: bool
+    """
+
+    # validate that the ttl file is in an acceptable directory
+    # rule: ttl needs to be in 3rd level -- aka 4th item in list['name', 'NIST', 'hypothesis', 'valid.ttl']
+    # rule: ttl needs to be in a subdirectory within the NIST directory
+    try:
+        if path_items[-3] == 'NIST' and len(path_items) == 4:
+            return True
+    except:
+        return False
+    return False
+
+def do_TA3_check(path_items):
```

Comment:
I created a submission in the following format

RUN_ID/NIST/{2 .ttl files}
RUN_ID/INTER-TA/{6 .ttl files}
RUN_ID/{2 .ttl files}

This fails. I would expect it to be a PARTIAL SUCCESS because there are two .ttl files in the RUN_ID folder. You can get this example in s3 at `aida-validation/m18_checker/Test_3.Test_3.Test_3a-Valid.zip`